src/Fixer/StringNotation/ExplicitStringVariableFixer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\StringNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class ExplicitStringVariableFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Convert implicit variables into explicit ones in double-quoted strings.',
```

Comment:
_"Convert**s**"_?

Reply:
Ok, but:
```
$ grep -rni Convert\  src/Fixer/
src/Fixer/StringNotation/HeredocToNowdocFixer.php:32:            'Convert `heredoc` to `nowdoc` where possible.',
src/Fixer/StringNotation/SingleQuoteFixer.php:32:            'Convert double quotes to single quotes for simple strings.',
src/Fixer/ClassNotation/NoPhp4ConstructorFixer.php:33:            'Convert PHP4-style constructors to `__construct`.',
```

Reply:
Those should be fixed as well IMO (not in this PR of course).